### DISCUSSION TITLE
Do you want to have a wiki describing every challenge and its solution and take away?

### DISCUSSION THREAD

**Discussion by commjoen** [OWASP/wrongsecrets#190] [Ideas]
We just nuked https://github.com/commjoen/wrongsecrets/issues/54 as we saw no need for filling the wiki: after all every answer can be found in the app. But is this something you as a user agree with? Or would you rather have a set of lessons and steps written down in the github wiki?
Give this an upvote if you want to have it! And with 100 upvotes, we will make sure you get it back ;-).

**Comment by djvinnie** [comment]
I wanted to discuss this with you @commjoen - Instead of having word-by-word complete solutions in the hint section, the hints themselves could be shortened and a separate write-up can be added elsewhere in the project! I am doing exactly this - I have about 7 challenges pending at this point and after that I can send my write-up over to you so you can provide feedback on it :)

**Reply by commjoen** [reply]
We received quite some praise of the self-containment of the app and given the short lessons and practical hints it was easy to maintain. We rather not support large write-ups **yet** until we have a large group of people wanting this. 
After all: maintaining documentation is a lot of work, and that is not what we intend to do at the moment.

**Reply by commjoen** [reply]
In the meantime: if you want to create write-ups, we can help you by reviewing any blogs you want to publish about the challenges. And if you want to contribute to OWASP projects in terms of write-ups : there are documentation projects that really like having contributions, such as:
https://cheatsheetseries.owasp.org/cheatsheets/Secrets_Management_Cheat_Sheet.html  and https://github.com/OWASP/wstg
